control-plane/csi-driver/src/bin/node/dev/nvmf.rs

```diff
@@ -335,3 +347,12 @@ pub(crate) fn volume_uuid_from_url(url: &Url) -> Result<Uuid, DeviceError> {
 
     extract_uuid(components[1]).map_err(|error| DeviceError::from(format!("invalid UUID: {error}")))
 }
+
+/// Extract nvmf fabric transport from Url.
+pub(crate) fn xprt_from_url(url: &Url) -> Result<String, DeviceError> {
```

Comment:
Let's try to use the TrType wherever possible? We can parse into the enum from the first caller?

Reply:
https://github.com/openebs/mayastor-control-plane/pull/858#discussion_r1753197929